app/javascript/packs/application/api.js

```diff
@@ -1,11 +1,17 @@
 import { create } from "apisauce";
 
+const getCSRFToken = () => {
+  if (typeof jest !== "undefined") {
```

Comment:
Is this logic to prevent CSRF checks when you're running tests?  Can you just mock this `getCSRFToken` function when you test instead?

Reply:
This is here because when tests are running, the `meta` tag won't be found in the document. Therefore, this function avoids querying the DOM in the test env.

Reply:
Mocking is probably a more perfect solution.

Reply:
> We the people, in order to mock a more perfect solution....
🇺🇸 🌎 🎆